virtual/vbox/vbox.go

```diff
@@ -392,6 +394,13 @@ func (lib *vBoxLibrary) getPathFromFile(file string) string {
 
 func (lib *vBoxLibrary) GetCopy(ctx context.Context, conf store.InstanceConfig, vmOpts ...VMOpt) (VM, error) {
 	path := lib.getPathFromFile(conf.Image)
+	// check whether provided image exists or not
+	// if not exits use default image kali.ova
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
+		// image does not exists, continue with default image
+		log.Debug().Msgf("Requested image [ %s ] cannot be found, continuing with default image [ %s ] ", conf.Image, defaultImage)
```

Comment:
Use fields in logging
Why have a default image? if no correct image send error and let user try again?

Reply:
I made some assumptions when doing it. Like I thought that `kali.ova` file is always exists, since it is as for as I know. However, that's right that assumptions are evil in most of the cases. ListFrontends function should list frontends which are configured on frontends.yml and under ova dir together. 

Furthermore, there might be the cases where ova file is corrupted and cannot be imported into the platform environment. In this case, there is a need to check whether provided ova file is corrupted or not. However, this issue itself might include more changes than it should. Therefore, my suggestion for integrity checking would be to do it in separate issue. 

So, currently if no correct image send > then kali ova will be chosen however if there is no kali.ova or corrupted one, then platform might crash. But this is another issue which is not directly related to this issue. 

This PR meant to solve issues like ;

- when create event  cmd send and there is no image configured as given from user then event information should not written to database until event is successfully up and running
- when non-exist frontend chosen, the platform should prefer default one > kali.ova
- when RAM/CPU values are not set correctly on frontends config file, the platform should prefer default values > 4096 MB , 1 CPU 

Of course, this PR is not cure to all issues it may happen but it heals some obvious ones.

Reply:
Im just trying to ensure that no inconvenient errors will happen if someone want to spin up their own instance of haaukins, it is totally right that at our deployment the kali.ova exist but it might not in other places.

I know this is not a part of the issue that this PR is trying to solve, however these changes is included here and therefore I chose to evaluate it as a part of this PR

Reply:
Just changed, so when provided frontend is not available on user side, program will check available frontends if there are any ova file available, platform will continue with first available file. 
- If provided ova file cannot be found go for first available frontend
- If frontend did not configured on frontends file, do not produce error instead run it with default values.
What are your feedbacks @Lanestolen & @Mikkelhost ?